src/components/ebay-snackbar-dialog/index.marko

```diff
@@ -0,0 +1,105 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+static var ignoredAttributes = [
+    "open"
+];
+
+class {
```

Comment:
Our convention is to have `component.js` if the class is pretty large. Please move this into that file, since this is quite a bit of logic.

Reply:
Maybe we could consider switching up that convention? I definitely have a preference for putting things in one file. It tends to make things easier to follow because you don't have to switch between files. 

Also trying to switch to a `component.js` is breaking and suddenly it is saying that `this.emit is not a function` which doesn't make a lot of sense.